molecule/default/tests/test_cinder_lvm.py

```diff
@@ -27,14 +31,36 @@ def test_cinder_lvs_volume_on_node(os_api_conn, host):
     Args:
         os_api_conn (openstack.connection.Connection): An authorized API
             connection to the 'default' cloud on the OpenStack infrastructure.
+        create_volume (def): A factory function for generating volumes.
+        openstack_properties (dict): OpenStack facts and variables from Ansible
         host (testinfra.host.Host): Testinfra host fixture.
     """
+    # create volume:
+    create_volume(
```

Comment:
You can capture the volume details by just assigning a variable to the `create_volume` call:
```suggestion
    test_volume = create_volume(
```